_data/projects/nativescript.yml

```diff
@@ -6,5 +6,5 @@ tags:
 - mobile
 - native
 upforgrabs:
-  name: up-for-grabs
-  link: https://github.com/nativescript/nativescript/labels/up-for-grabs
+  name: help wanted
+  link: https://github.com/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+user%3ANativeScript
```

Comment:
You can simplify this to `https://github.com/nativescript/nativescript/labels/help%20wanted`

Reply:
I want to search issues in the whole NativeScript organization, not only the NativeScript repo inside it.

I know that `NativeScript` repo inside `NativeScript` organization is sometimes confusing :)

Reply:
Aha, I missed that. Apologies!